databricks/koalas/tests/test_dataframe.py

```diff
@@ -1222,9 +1227,13 @@ def test_transpose(self):
                                                              ('rg2', 'z')]))
         kdf3 = ks.from_pandas(pdf3)
 
-        self.assertEqual(
-            repr(pdf3.transpose().sort_index()),
-            repr(kdf3.transpose(limit=None).sort_index()))
+        try:
+            set_option("compute.max_rows", None)
```

Comment:
set_option("compute.max_rows", None) should be out of this try block to be consistent with other codes you're fixing.